src/formatDate.js

```diff
@@ -50,7 +50,56 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const oldDateArr = date.split(fromFormat[3]);
+  const newDateArr = [];
+  const objDate = {};
+
+  fromFormat.forEach((el, ind) => {
+    switch (true) {
+      case el.includes('YY'):
+        objDate.year = oldDateArr[ind];
+        break;
+      case el.includes('MM'):
+        objDate.month = oldDateArr[ind];
+        break;
+      case el.includes('DD'):
+        objDate.day = oldDateArr[ind];
+        break;
```

Comment:
```suggestion
    switch (el) {
      case 'YY':
      case 'YYYY':
        objDate.year = oldDateArr[ind];
        break;
      case 'MM':
        objDate.month = oldDateArr[ind];
        break;
      case 'DD':
        objDate.day = oldDateArr[ind];
        break;
```
let's use `switch` in a more common way, now it's more like the set  of `if` blocks
same below

Reply:
Change switch to if